recipes/lightning-flash/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "lightning-flash" %}
+{% set version = "0.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lightning-flash-{{ version }}.tar.gz
+  sha256: d3f44d89b800113cb86aa0dedf088ef30a1f21cfe52bbc8e90a57d5aa4c7c56c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy
+    - packaging
+    - pandas <1.3.0
+    - pydeprecate
+    - python >=3.6
+    - pytorch-lightning >=1.3.1
+    - pyyaml >=5.1
+    - pytorch >=1.7
+    - torchmetrics
+    - tqdm
+
+test:
+  imports:
+    - flash
+    - flash.core
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/PyTorchLightning/lightning-flash
+  summary: Flash is a framework for fast prototyping, finetuning, and solving most standard deep learning challenges
+  license: Apache-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - AddYourGitHubIdHere
```

@Borda and @ethanwharris, are you both ok to be listed as maintainers for the lightning-flash conda recipe? I can also add others in.

```suggestion
    - Borda
    - ethanwharris
    - weiji14
```